src/lua-function-list.json

```diff
@@ -13,15 +13,15 @@
     "addWordToDictionary": "addWordToDictionary(word)",
     "adjustStopWatch": "adjustStopWatch(watchID/watchName, amount)",
     "alert": "alert([seconds])",
-    "ansi2decho": "ansi2decho(text, default_colour)",
```

Comment:
Not sure why this and others below are being marked as removed. will try to take a look in a moment

Reply:
I forgot to remove the 'merged' part of the subtitle when copying over from Area_51